### PR TITLE
Add area/dependencies to bom and build-parent changes

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -19,9 +19,7 @@ pr:
       - LICENSE.txt
       - dco.txt
       - .github/ISSUE_TEMPLATE/*.md
-      - .github/labeler.yml
       - .github/boring-cyborg.yml
-      - .github/workflows/*
```
Comment (maxandersen) [inline azure-pipelines.yml]: why are you removing that one? it will show up eventually :)

**Review by maxandersen — CHANGES_REQUESTED** [review]
+1 on the label dir changes but -1 on azure pipelines.

File: azure-pipelines.yml (hunk 1 above)
Comment (gastaldi) [reply azure-pipelines.yml]: Ah I thought you replaced with the boring cyborg one. I'll revert that change

File: .github/boring-cyborg.yml [hunk 2]
```diff
@@ -49,7 +49,6 @@
 ### area/build
 ### area/camel
 ### area/core
-### area/dependencies
```
Comment (maxandersen) [inline .github/boring-cyborg.yml]: added that just to clean it up

**Review by maxandersen — APPROVED** [review]
Golden LGTM!

**Comment by gastaldi** [comment]
Heh, I was wondering how that comment got away automatically.

**Pull request merged**